stratum/hal/lib/tdi/tdi_sde_helpers.cc

```diff
@@ -444,22 +459,28 @@ ::util::Status GetAllEntries(
       RETURN_IF_TDI_ERROR(table->dataAllocate(&data[i]));
       pairs.push_back(std::make_pair(keys[i].get(), data[i].get()));
     }
-    uint32 actual = 0;
+    actual = 0;
     RETURN_IF_TDI_ERROR(table->entryGetNextN(
         *tdi_session, tdi_dev_target, flags, *(*table_keys)[0], pairs.size(),
         &pairs, &actual));
 
-    table_keys->insert(table_keys->end(), std::make_move_iterator(keys.begin()),
-                       std::make_move_iterator(keys.end()));
+    auto keys_end=keys.begin();
```

Comment:
Coding guidelines: insert a single space before and after the binary operator `=`.